packages/opencensus-exporter-zipkin/test/test-zipkin.ts

```diff
@@ -155,7 +155,7 @@ describe('Zipkin Exporter', function() {
           'localEndpoint': {'serviceName': 'opencensus-tests'},
           'name': 'spanTest',
           'parentId': rootSpan.id,
-          'shared': true,
+          'shared': false,
```

Comment:
Add a test for when `shared` should be true.

Reply:
Currently, it has one bigger test that contains both root and child span translation and covers both cases of shared. `shared` is not the only difference, but I guess we could break it down to separate root and child span tests.

Reply:
I separated them